release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -84,6 +84,13 @@ The {product-title} 4.10 installer uses new default component types for installa
 * AWS EC2 M6i instances for both control plane and compute nodes, where available
 * AWS EBS gp3 storage
 
+[id="ocp-4-10-enhancements-to-API-in-installconfig"]
+==== Enhancements to API in install.config
```

Comment:
* in -> in the
* install.config -> install-config? or `install-config.yaml` file?

Reply:
in the `install-config.yaml` file. Updated as suggested.